src/__tests__/Input.spec.js

```diff
@@ -63,6 +63,12 @@ describe('Input', () => {
     expect(wrapper.text()).toBe('Yo!');
   });
 
+  it('should render without children when type is "textarea"', () => {
+    const wrapper = shallow(<Input type="textarea">Yo!</Input>);
+
+    expect(wrapper.text()).toBe('');
```

Comment:
Shouldn't the test be checking to ensure that 'Yo!' is included inside the textarea. This test doesn't cover that.

Reply:
Currently, that is how it will work (and react will spit out a warning), but in react 17 it will not work. React wants people to use `value`/`defaultValue` even for `<textarea>`, making `<textarea />` a void element.
This becomes a breaking change for reactstrap now, ahead of the breaking change react 17 will force us to make. It simplifies things a little it makes `<textarea>` and `<input>` basically the same.... which IMHO, HTML should have done `<input type="textarea" />` (I wonder why they didn't... I'll have to Google)